components/molecules/AccessabilityMenu.js

```diff
@@ -23,6 +23,12 @@ export default function AccessabilityMenu(props) {
     function contrast(directionMultiplier) {
         let newContrastLevel = getNewLevel(directionMultiplier, contrastLevel);
         setContrastLevel(newContrastLevel);
```

Comment:
Peut-être qu'il pourrait faire le set juste si c'est dans les limites ? Du genre:
```suggestion
        if(newContrastLevel>=20 && newContrastLevel<=180){
            setContrastLevel(newContrastLevel);
        }
```